src/main/java/pl/ergohestia/ehj1/ivesta/entities/Route.java

```diff
@@ -34,4 +39,12 @@
 
     @Column(name = "transport_volume")
     private Integer transportVolume;
+
+    public Route(String startAddress, String destinationAddress, Integer routeLength, TransportType transportType, Integer transportVolume) {
```

Comment:
Zrób jednolicie. Skoro używasz `@NoArgsConstructor` to jest też analogiczna `@AllArgsConstructor`, która wygląda, że zadziała dokładnie tak, jak to tutaj zadeklarowałeś. A może nawet zamiast takiego tasiemca, użyć od razu buildera (`@Builder`)?

Reply:
@AllArgsConstructor wrzuciłby do konstruktora id, którego nie używam w deklaracji